Cargo.lock

```diff
@@ -188,19 +179,36 @@ checksum = "4785bdd1c96b2a846b2bd7cc02e86b6b3dbf14e7e53446c4f54c92a361040822"
 
 [[package]]
 name = "clap"
-version = "2.33.1"
+version = "3.0.0-beta.1"
```

Comment:
`2.33.1` is using some deprecated APIs in `crate_authors!` macro. According to [this comment](https://github.com/clap-rs/clap/pull/1853#issuecomment-618499296) it's not going to be fixed in the version `2` (which is not actively maintained anymore) — and I really want my `clippy -D warning`.
This is especially important since we're going to mostly maintain this tool in "hands off" mode, mostly relying on Dependabot and automation to keep our dependencies up to date. Since eventually we would have to update to `3.0` anyway, it's better to do that now, while we still have context of the task in heads — and then enjoy only having to occasional click "merge" on a green PR afterwards.